README.md

```diff
@@ -56,7 +56,7 @@ const target = {
 
 ### Evaluate the flag with default value set to false
 ```
```

Comment:
Not 100% required, but GitHub will give better highlighting if you tell it what language your code is in.
```suggestion
```typescript
```